bin/n

```diff
@@ -740,8 +740,7 @@ activate() {
       printf '\nNote: the node command changed location and the old location may be remembered in your current shell.\n'
       log old "${original_node}"
       log new "${active_node}"
-      # shellcheck disable=SC2016
```

Comment:
@shadowspawn I noticed this shellcheck exception does not seem to apply anymore, so I removed it as well.

Reply:
Yes, that was due to a warning about the `$PATH` inside single quotes, which is gone.